helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -120,11 +121,21 @@ internal class ListLicensesCommand : CommandWithHelp() {
 
     override fun runCommand(jc: JCommander): Int {
         var ortResult = ortResultFile.readValue<OrtResult>()
+        if (ortResult.getPackageOrProject(packageId) == null) {
+            println("Could not find a package for the given id `$packageId`.")
+            return -1
```

Comment:
Exit codes should never be negative integers as at least [POSIX only considers the 8 least significant bits](https://stackoverflow.com/a/47266617/1127485). Mind fixing this up @fviernau?

Reply:
Sure, I'll fix it. Thanks for pointing that out!